docs/02-app/02-api-reference/05-next-config-js/serverComponentsExternalPackages.mdx

```diff
@@ -20,26 +20,42 @@ module.exports = nextConfig
 
 Next.js includes a [short list of popular packages](https://github.com/vercel/next.js/blob/canary/packages/next/src/lib/server-external-packages.json) that currently are working on compatibility and automatically opt-ed out:
```

Comment:
Since this file is mdx, perhaps we should import this json file so this list below is always up to date